cmd/soroban-cli/src/config/secret.rs

```diff
@@ -140,6 +153,25 @@ impl Secret {
     }
 }
 
+pub enum StellarSigner {
```

Comment:
I propose to rename `StellarSigner` -> `SingerMethod` and `Stellar` -> `StellarSigner` for clarity

Reply:
Well `Stellar` is in the signer module, so it's a little redundant. We could instead do `signer::Stellar` when referencing it so that it's clear. But clippy doesn't like when the module name is used in a name.

But I do agree with `StellarSigner` being a poor name. And `Stellar` for that matter. @leighmcculloch Opinions?  Naming is the worst!